x-pack/plugins/alerts/server/alert_type_registry.ts

```diff
@@ -32,7 +32,7 @@ export interface ConstructorOptions {
 
 export interface RegistryAlertType
   extends Pick<
-    NormalizedAlertType,
+    UntypedNormalizedAlertType,
```

Comment:
We can't have mixed types in the `Map`, so we have to string these when we _set_ them, and rely on the user to provide the right type when they _get_ them.